docs/configs/html/conf.py

```diff
@@ -177,6 +177,10 @@
     'ledger-api-test-tool-maven': ('https://repo1.maven.org/maven2/com/daml/ledger-api-test-tool/{}/%s-{}.jar'.format(version, version), None)
 }
 
+rst_prolog = """
+.. _installer: https://github.com/digital-asset/daml/releases/download/v{}/daml-sdk-{}-windows.exe
+""".format(release, release)
```

Comment:
Can you use `release=release`?

(I don't know if the following will work.)

```suggestion
rst_prolog = """
.. _installer: https://github.com/digital-asset/daml/releases/download/v{release}/daml-sdk-{release}-windows.exe
""".format(release=release)
```

Reply:
This is plain Python, so probably, yes.

Reply:
Tested, does seem to work. Thanks!

Reply:
I did not manage to get the simple substitution to work in a URL (works fine in text though). Happy to switch to that if you can make it work, but the `rst_prolog` thing is the only way I could make it work.